Create April Fools' day article

MilkyIQ opened the pull request:
~~**Do not merge this pr until after 1 April 2021, for obvious reasons**~~ the section for 2021 has been added; this pr should be ready to merge after final review

Creates a new article that details the complete history (2009–2020) of osu!'s April Fools' day jokes/pranks. The article still needs *some* official official verification in a few articles, but I don't really expect those to be untrue and more just need fact-checking.

**Follows the conventions of citation detailed in #4911**; if you have any objections towards any aspect of the method of citing sources used here, please comment on the aforementioned issue instead of this pr. Resolves #117

Joehuu commented on wiki/History_of_osu!/April_fools/en.md:
```diff
@@ -0,0 +1,324 @@
+---
```
Folder name should be `April_Fools`, as it is a proper noun.

Note: there is one img already in `April_Fools`.

MilkyIQ replied:
Hmm, that's a little weird, because folder is already capitalised. Not sure why its displaying as lowercase here

Joehuu replied:
Are you talking about your local repo? Try doing `git config core.ignorecase false` in your local repo dir.

MilkyIQ replied:
seemed to work, thank you!

Joehuu commented:
This is still in a bad state (there are duplicate files now).

MilkyIQ commented:
> This is still in a bad state (there are duplicate files now).

@Joehuu This *should* be fixed now. I believe this was just an oversight by me (renaming the files at some point broke git because git is case-sensitive by default).

I deleted the folder on-website (yeah yeah ik), git pulled, and then discarded any changes that attempted to remove the files from `/April_Fools/` (yes yes ik but git rm was being finnicky with me). I'm not 100% sure if this is a fool-proof solution, though, so please do let me know if I've majorly fluffed something up.

zeusminus commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+### Free osu! supporter tags
+
+On 1 April 2016 many osu! players were surprised to discover that they had suddenly and inexplicably received an [osu! supporter tag](https://osu.ppy.sh/home/support) despite never having bought or being gifted one. The supporter tag that was given to players was fully functional and acted just like a regular supporter tag, however the change was rolled back the next day \[1,2,3,4,5,6,7\].
```
```suggestion
On 1 April 2016 many osu! players were surprised to discover that they had suddenly and inexplicably received an [osu! supporter tag](https://osu.ppy.sh/home/support) despite never having bought or being gifted one. The supporter tag that was given to players was fully functional and acted just like a regular supporter tag; however, the change was rolled back the next day \[1,2,3,4,5,6,7\].
```

zeusminus commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+### Sneezing girl sound effect (2019)
+
+For April Fools' day 2019 there was approximately a 1 in 20 chance of hearing the sound effect of a high-pitched girl sneezing upon opening a [beatmap](/wiki/Beatmap) \[1,2,3\].
```
```suggestion
For April Fools' day 2019 there was approximately a 1 in 20 chance of hearing the sound effect of a high-pitched girl sneezing upon opening a beatmap \[1,2,3\].
```

zeusminus commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+## 2020
+
+### MillhioreF joins the Featured Artists
```
There needs to be noted somewhere down that [Waltz o' the Irish](https://osu.ppy.sh/beatmapsets/73348) was moved to the Loved section that day as part of the April Fools' joke.

MilkyIQ replied:
I'll look through my sources, but do you have any direct proof of this?

Also assuming that it was moved back out of Loved the same day?

zeusminus replied:
I didn't bother providing you with any source to this, since it's already noted on the newspost. It is still in Loved section, wasn't moved back out.

MilkyIQ replied:
Oh, totally my bad lol. This just snuck past me ig

zeusminus commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+### MillhioreF joins the Featured Artists
+
+[MillhioreF](https://osu.ppy.sh/users/941094)—a long-time osu! moderator, developer and [Easy mod](/wiki/Game_modifier/Easy) player—was announced in [a newspost](https://osu.ppy.sh/home/news/2020-04-01-new-featured-artist-millhioref) to have "joined" the list of [Featured Artists](/wiki/Featured_Artists) as "Millhiore Firianno Biscotti" on 1 April 2020 with a selection of five songs to boot \[1\] ...
```
You can also add a detail about millhi's featured artist ID, which was 727, though the page no longer exists.

![](https://zeu.s-ul.eu/awYdE8En.png)

MilkyIQ replied:
Ehhh. I typically enjoy added bits of trivia but this is just a bit too trivial imo. If anyone is really in dire need of this info they'll know how to find it

zeusminus commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+*For the complete history of osu!, see: [History of osu!](/wiki/History_of_osu!)*
+
+Every year, the osu! developers like to play pranks on the community on April Fools' day. This article contains a list of all the April Fools' day jokes that were played on the osu! community since 2009.
```
```suggestion
Every year, the [osu!team](/wiki/People/The_Team) likes to play pranks on the community on April Fools' day. This article contains a list of all the April Fools' day jokes that were played on the osu! community since 2009.
```

zeusminus reviewed (commented):
Aside from my suggestions below, I would like to say a few things regarding references. First of all, as much as I liked the way you handled them, I personally think that it will hinder the readability of the document to some extent. As an alternative, I highly recommend going with either one of the following methods:
- **List all the references at the end of the article.** Since this is how Wikipedia, and many other documentations, handle the references/citations, we should be doing the same in order not to dissect the article into pieces with several references sections. You can allocate numbers for each reference with a footnote, and these footnotes can link to the References section at the end of the article. It might be troublesome to find a certain citation amongst many of them though since it's impossible to directly link to a particular citation.
- **Cite them via in-line notes (`<!--text-->`).** This method was used in [Mapping and modding timeline](https://github.com/ppy/osu-wiki/blob/master/wiki/Mapping_and_Modding_Timeline/en.md), and I personally think it is the best method to cite things without hindering both readability of the article and searchability of the references. The only downside I could think of is that they are not so easily reachable, unlike Wikipedia, if one does not know GitHub exists and is looking for citations.

This very question of how information should be cited is something that we should resolve, obviously. However, for the time being, we have to utilize what we currently have.

The contents of the article are well-documented and very informative, I have nothing contrary to say here.

TPGPL commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+In short, in order to play or restart a [beatmap](/wiki/Beatmap), users must spend one osu!coin, and once they ran out of osu!coins, they would either have to stop playing and wait until the next day, or pay real money to get more osu!coins. In spite of this description, however, actual gameplay was not affected, and users could just continue playing as normal once all their osu!coins were deplet ...
+
+A new osu! main screen was also made that had a barrage of slowly rising osu!coins in the background, along with a slightly changed main theme, in which the exclamation "circles!" was replaced with a robotic-sounding "and buy the coins" before the beat drop. Additional new textures, sound effects, animations, and music were created for this prank, including a counter that would display the amount ...
```
```suggestion
The main screen of osu! also had a barrage of slowly rising osu!coins in the background, along with a slightly changed main theme, in which the exclamation "circles!" was replaced with a robotic-sounding "and buy the coins" before the beat drop. Additional new textures, sound effects, animations, and music were created for this prank, including a counter that would display the amount of coins players had throughout their plays. \[5,6\]. <!--needs further verification; also not sure if this paragraph is necessary or not-->
```

TPGPL commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+### osu! in virtual reality
+
+On 1 April 2016, [a newspost](https://osu.ppy.sh/home/news/2016-04-01-oculus-rift-to-be-supported-as-an-input-method) was published detailing the announcement of plans to add support for the [Oculus Rift](https://en.wikipedia.org/wiki/Oculus_Rift "Wikipedia") as a new [input method](/wiki/Gameplay/Input_device) in osu!. The post, written by [Evrien](https://osu.ppy.sh/users/791660), referenced ma ...
```
```suggestion
On 1 April 2016, [a news post](https://osu.ppy.sh/home/news/2016-04-01-oculus-rift-to-be-supported-as-an-input-method) was published detailing the announcement of plans to add support for the [Oculus Rift](https://en.wikipedia.org/wiki/Oculus_Rift "Wikipedia") as a new [input method](/wiki/Gameplay/Input_device) in osu!. The post, written by [Evrien](https://osu.ppy.sh/users/791660), referenced many quotes from a supposed interview of [peppy](https://osu.ppy.sh/users/2) in which he explained his reasoning for the announcement and ideas for how the concept might work \[1\].
```

sr229 commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+"osu!core" is the name given to osu!'s 2011 April Fools' day prank. The prank caused every single [beatmap](/wiki/Beatmap)'s audio to be pitched and sped up in the style of [Nightcore](https://en.wikipedia.org/wiki/Nightcore "Wikipedia") remixes. Although just an April Fools' day prank, it later became a reality when the [Nightcore mod](/wiki/Game_modifier/Nightcore) was introduced as an actual p ...
+
+#### References
```
Instead of creating seperate references here, use one big references section at the bottom. We want everyone to focus on the article content firsthand.

TPGPL commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+On 1 April 2016, [a newspost](https://osu.ppy.sh/home/news/2016-04-01-oculus-rift-to-be-supported-as-an-input-method) was published detailing the announcement of plans to add support for the [Oculus Rift](https://en.wikipedia.org/wiki/Oculus_Rift "Wikipedia") as a new [input method](/wiki/Gameplay/Input_device) in osu!. The post, written by [Evrien](https://osu.ppy.sh/users/791660), referenced ma ...
+
+Regarding how players might use the Oculus Rift as an input method, the newspost describes that "the player will experience a first person point-of-view of the cursor as it travels to and from hit-objects on screen…" and will hit hit objects by "…having the player produce vowel-like sounds using their mouth". No actual in-game changes were made relating to using the Oculus Rift or a similar [virt ...
```
```suggestion
Regarding how players might use the Oculus Rift as an input method, the news post describes that "the player will experience a first person point-of-view of the cursor as it travels to and from hit-objects on screen…" and will hit hit objects by "…having the player produce vowel-like sounds using their mouth". No actual in-game changes were made related to using the Oculus Rift or a similar [virtual reality (VR)](https://en.wikipedia.org/wiki/Virtual_reality "Wikipedia") device to control osu! \[1\].
```

TPGPL commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+### Dancing Auto mod cursor/dancing pippi
+
+"dancing pippi" (also known as "dancing Auto mod cursor") is the nickname given to one of osu!'s April Fools' jokes for 2016 in which an update was released that caused the game cursor in [Auto](/wiki/Game_modifier/Auto) mod [replays](/wiki/Replay) to circle around the current [hit object](/wiki/Hit_object) in pixel-perfect fashion before eventually hitting the object right on time, which is in c ...
```
```suggestion
"dancing pippi" (also known as "dancing Auto mod cursor") is the nickname given to one of osu!'s April Fools' jokes for 2016 in which an update was released that caused the game cursor in [Auto](/wiki/Game_modifier/Auto) mod [replays](/wiki/Replay) to circle around the current [hit object](/wiki/Hit_object) in pixel-perfect fashion before eventually hitting the object right on time, in contrast to the Auto mod's usual robotic and perfectly straight movements. The aforementioned update was rolled back in a subsequent one the next day \[1,2,3,4\]. <!--still needs official official verification-->
```

sr229 reviewed (changes requested):
Just a major nit

MilkyIQ commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+### "Lemon Tree" gets ranked
+
+The [beatmap](/wiki/Beatmap) "Best of No.1 Hits - Lemon Tree (MillhioreF)" was [ranked](/wiki/Beatmap/Category#ranked) on 1 April 2009 as part of that year's April Fools' joke. The beatmap was very much an in-joke within the community with users sarcastically claiming that such a beatmap should be kept ranked forever \[1,2,3]\. It's ranked status was removed by moderators shortly afterwards \[2\] ...
```
assuming you meant 
```suggestion
The [beatmap](/wiki/Beatmap) ["Best of No.1 Hits - Lemon Tree (MillhioreF)"](https://osu.ppy.sh/beatmapsets/57878#osu/174267) was [ranked](/wiki/Beatmap/Category#ranked) on 1 April 2009, as part of that year's April Fools' joke. The beatmap was very much an in-joke within the community with users sarcastically claiming that such a beatmap should be kept ranked forever \[1,2,3\]. Its ranked status was removed by the moderation team shortly afterwards \[2\].
```

MilkyIQ commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+### touhosu!
+
+For the 2010 April Fools' prank, the osu! website along with the in-game main menu screen was changed to theme with characters and references from the [Touhou Project](https://en.wikipedia.org/wiki/Touhou_Project "Wikipedia") (shown below). The changes included the addition of the character Marisa Kirisame and a display of differently-coloured butterflies in a circular pattern across the main men ...
```
this was intentional; using "theme" as a verb here

zeusminus replied:
ah, didn't notice. both forms seem to be applicable here though.

MilkyIQ commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+#### References
+
+1. [Bad Apple!! Ranking Chart, 4 April 2012](https://web.archive.org/web/20120404051938/http://osu.ppy.sh/p/chart?ch=BADAPPLE "Internet Archive — osu!")
+2. [osu! newspost [Cyclone] (1 April 2012) - "Bad Apple!! Ranking Chart"](https://osu.ppy.sh/community/forums/topics/79128?start=1431905)
```
both of these are admin forbidden, but if im not mistaken isn't 1431905 the number attributed to start and not the post number?

zeusminus replied:
that is the permalink that goes to the forum post specifically, rather than the topic. both redirect to the same page and same post, but my suggestion contains a clearer one

MilkyIQ commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+#### References
+
+1. [Reddit comment (r/osugame) [u/Saluya_Lv9] - "April 1st"](https://www.reddit.com/r/osugame/comments/2201so/april_1st/ "Reddit")
+2. [osu! forum comment (Off-Topic) [Jazz] - "Your prediction of osu! April Fools"](https://osu.ppy.sh/community/forums/topics/125391?start=2215004)
+3. [osu! forum comment (Off-Topic) [Brian OA] - "Your prediction of osu! April Fools"](https://osu.ppy.sh/community/forums/topics/125391?start=2215194)
+4. [Tweet [@little_2d] - 27 June 2019](https://twitter.com/little_2d/status/1144316731407683584 "Twitter")
+5. [osu! forum comment (Development) [kingking9] - "osu! Community Localisation Project"](https://osu.ppy.sh/community/forums/topics/104342?start=2342998)
+6. [osu! forum comment (Development) [peppy] - "osu! Community Localisation Project"](https://osu.ppy.sh/community/forums/topics/104342?start=2343044)
```
I mainly left these citations in as proof of hearsay. It's very unlikely that a group of seemingly unrelated accounts across different forums would come together to spread some malicious false info about some random april fools' joke from 2013.

Plus, the only citation that is "definitive proof" that this joke did exist was the peppy comment, and even that could be questionable as the comment didn't mention april fools' directly and is only assuming that "ts" is short for tsundere

MilkyIQ commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+![](img/2015-osucoins.png "Three osu! coins")
+
+On 31 March 2015, [peppy](https://osu.ppy.sh/users/2) released [a newspost](https://osu.ppy.sh/home/news/2015-03-31-osucoins) that announced the addition of a new in-game currency to be added to osu!, dubbed "osu!coins" \[1,2\]. The newspost explained what the in-game currency was and how it worked along with a specially-created [osu!academy video](https://www.youtube.com/watch?v=BImc5McuK1o "You ...
```
oh god have i been spelling it "newspost" this entire time? where did i get that?

ill add a fix for this in a separate commit with citation fixes

MilkyIQ commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,324 @@
+
+*Notice: McOsu is developed separately from and has no direct affiliation with osu! or ppy Pty Ltd.*
+
+However, even though the official osu! developers have/had no intention of genuinely adding VR support, the idea of osu! in VR piqued interest in the minds of some fans. This interest eventually culminated into an unofficial fan project that was started shortly afterwards that aimed to create a free and open source client to practice osu! [beatmaps](/wiki/Beatmap) in with more features and [game  ...
```
i had searched around and it seemed that "open source" was the more commonly favoured spelling

MilkyIQ commented:
> Aside from my suggestions below, I would like to say a few things regarding references. First of all, as much as I liked the way you handled them, I personally think that it will hinder the readability of the document to some extent. As an alternative, I highly recommend going with either one of the following methods:
> 
> * **List all the references at the end of the article.** Since this is how Wikipedia, and many other documentations, handle the references/citations, we should be doing the same in order not to dissect the article into pieces with several references sections. You can allocate numbers for each reference with a footnote, and these footnotes can link to the References section at the end of the article. It might be troublesome to find a certain citation amongst many of them though since it's impossible to directly link to a particular citation.

My only issue I have with citing them all at the bottom is just the sheer number of sources. Wikipedia can get away with this because they have mediawiki and you can click on the in-line citation to go straight to the source. But with markdown, users would have to scroll aaaalll the way down (probably forgetting where they were in the process) and then scroll all the way back up.

Again though, that's just my qualm with it, but it others think references at the bottom is worth it, I'll go ahead and change that. *(I'll create a separate commit so I can revert the change back if needed)*

> * **Cite them via in-line notes (`<!--text-->`).** This method was used in [Mapping and modding timeline](https://github.com/ppy/osu-wiki/blob/master/wiki/Mapping_and_Modding_Timeline/en.md), and I personally think it is the best method to cite things without hindering both readability of the article and searchability of the references. The only downside I could think of is that they are not so easily reachable, unlike Wikipedia, if one does not know GitHub exists and is looking for citations.
> 
> This very question of how information should be cited is something that we should resolve, obviously. However, for the time being, we have to utilize what we currently have.

I'd go with option 1 over this imo

zeusminus commented:
> But with markdown, users would have to scroll aaaalll the way down

Well, not really. All footnotes will link to the References section. See the following:

```
## 2012 april fools

this is the information and here's the footnote [\[1\]](#references)

---

## References

[1.](#2012-april-fools) this is a reference
```
This way, users will not have to scroll up and down repeatedly. Both the information and the citation will remain one-click away!

---

By the way, I just discovered another April Fools' joke that occurred in 2010. There is [a forum thread in Off-Topic](https://osu.ppy.sh/community/forums/topics/27552) that contains a link to an admin-only subforum named "Community Development (AprilFools')", which is supposed to remain *confidential*. It contains a few threads regarding joke projects like purchasing beatmaps using kudosu, cash incentives for quality mod posts, and shutting down the online aspect of the game for non-supporters. 

I'm not sure if the subforum is still visible for normal users, but I can provide you with further details if you're considering adding this piece of information.

MilkyIQ commented:
> > But with markdown, users would have to scroll aaaalll the way down
> 
> Well, not really. All footnotes will link to the References section. See the following:
> 
> ```
> ## 2012 april fools
> 
> this is the information and here's the footnote [\[1\]](#references)
> 
> ---
> 
> ## References

> [1.](#2012-april-fools) this is a reference
> ```
> 
> This way, users will not have to scroll up and down repeatedly. Both the information and the citation will remain one-click away!

Actually not a bad idea. I'll add it soon. Not sure if I'll add that second bit though, as that sounds like it'd be a trip to find-where-you-made-one-+little-mistake-that-broke-everything-ville

> 
> By the way, I just discovered another April Fools' joke that occurred in 2010. There is [a forum thread in Off-Topic](https://osu.ppy.sh/community/forums/topics/27552) that contains a link to an admin-only subforum named "Community Development (AprilFools')", which is supposed to remain _confidential_. It contains a few threads regarding joke projects like purchasing beatmaps using kudosu, cash incentives for quality mod posts, and shutting down the online aspect of the game for non-supporters.
> 
> I'm not sure if the subforum is still visible for normal users, but I can provide you with further details if you're considering adding this piece of information.

The more the merrier! I'd definitely be willing to take a look

zeusminus commented:
> that sounds like it'd be a trip to find-where-you-made-one-+little-mistake-that-broke-everything-ville

I fail to understand why you think like that. Doing so would be building a two-way road between the information and the citation to ensure ease of access. If anything breaks, there's always a way to fix it.

> The more the merrier! I'd definitely be willing to take a look

Cool! Hit me up if the subforum is inaccessible to you, along with the information you need and I'll be happy to provide you with it.

zeusminus commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,283 @@
+- A Waltz From The Geographical Region Known as Ireland but Also as Éire
+- There's Gold Beneath Your Waltzing Rainbow (feat. Mismagius)
+
+["MillhioreF - Waltz o' the Irish (MillhioreF)"](https://osu.ppy.sh/beatmapsets/73348#osu/326585)—a [long-running joke beatmap](#2009) within the community—was also Loved on 31 March 2020 as part of this joke.
```
I don't see any particular relation between this map and the one for the April Fools' 2009, except for both being mapped by MillhioreF. The way you worded it gives the impression that Lemon Tree and Waltz o' the Irish are the same beatmap.
```suggestion
["MillhioreF - Waltz o' the Irish"](https://osu.ppy.sh/beatmapsets/73348#osu/326585) was also [Loved](/wiki/Beatmap/Category#loved) on 31 March 2020 as part of this joke.
```

MilkyIQ replied:
this was just me being stupid; I misread the title

MilkyIQ commented:
Leaving this comment here for anyone returning for a last review: I realized there was a duplicate citation in te references list (14 and original 33), so I removed the citation and updated the citations accordingly. This was done by hand with ctrl-f, so I'm *pretty sure* there are no errors, but you can double check for yourself. If you come back to see what seems like a lot of changes, that's why.

sr229 reviewed (approved):
Nothing outstanding in my skimming, LGTM

TPGPL commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,294 @@
+
+— Cyclone, "Bad Apple Ranking Chart!" [[21,22]][r]
+
+The chart was opened on 1 April 2012 and closed on 2 April 2012. At the end of the ranking period, player [Mesita](https://osu.ppy.sh/users/201459) was the number one player with a ranked score of 145,623,328 [[23]][r].
```
```suggestion
The chart was opened on 1 April 2012 and closed on 2 April 2012. At the end of the ranking period, [Mesita](https://osu.ppy.sh/users/201459) was the number one player with a ranked score of 145,623,328 [[23]][r].
```

TPGPL commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,294 @@
+
+### BanchoBot becomes a tsundere
+
+On 1 April 2013, [BanchoBot](/wiki/BanchoBot) was turned into a [tsundere](https://en.wikipedia.org/wiki/Tsundere "Wikipedia"). On this April Fools' day, any time a user would issue a command to BanchoBot or prompt it to speak in a public chat, its chat messages would be replaced with stereotypical tsundere-like responses to a romantic interest [[24,25,26,27,28,29]][r].
```
```suggestion
On 1 April 2013, [BanchoBot](/wiki/BanchoBot) was turned into a [tsundere](https://en.wikipedia.org/wiki/Tsundere "Wikipedia"). On this April Fools' day, every time a user would issue a command to BanchoBot or prompt it to speak in a public chat, its chat messages would be replaced with stereotypical tsundere-like responses to a romantic interest [[24,25,26,27,28,29]][r].
```

TPGPL commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,294 @@
+
+*See also: [osu!coin](/wiki/osu!coin)*
+
+![](img/2015-osucoins.png "Three osu! coins")
```
```suggestion
![](img/2015-osucoins.png "The osu! coins")
```

TPGPL commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,294 @@
+
+In short, in order to play or restart a [beatmap](/wiki/Beatmap), users must have spent one osu!coin, and once they ran out of osu!coins, they would either have to stop playing and wait until the next day, or pay real money to get more osu!coins. In spite of this description, however, actual gameplay was not affected, and users could just continue playing as normal, even if all their osu!coins we ...
+
+The main screen of osu! also had a barrage of slowly rising osu!coins in the background, along with a slightly changed main theme, in which the exclamation "circles!" was replaced with a robotic-sounding "and buy the coins" before the beat drop. Additional new textures, sound effects, animations, and music were created for this prank, including a counter that would display the amount of coins pla ...
```
```suggestion
The main screen of osu! also had a barrage of slowly rising osu!coins in the background, along with a slightly changed main theme, in which the exclamation "circles!" was replaced with a robotic-sounding "and buy the coins" before the beat drop. Additional new textures, sound effects, animations, and music were created for this prank, including a counter that would display the amount of coins players had throughout their plays. [[37,38]][r]. <!--needs further verification-->
```

Pachiru commented on wiki/History_of_osu!/April_Fools/en.md:
```diff
@@ -0,0 +1,294 @@
+- [Masayoshi Minoshima feat. nomico - Bad Apple!! (ouranhshc)](https://osu.ppy.sh/beatmapsets/18260)
+- [Spiritsoulxx - Bad Apple!! (Tony)](https://osu.ppy.sh/beatmapsets/23760)
+- [Kommisar - Bad Apple!! (Chiptune Ver.) (Sushi)](https://osu.ppy.sh/beatmapsets/28222)
+- [Kalafina - Bad MagiApple (Makar8000)](https://osu.ppy.sh/beatmapsets/32003)
```
There is this one too https://osu.ppy.sh/beatmapsets/299000#osu/670892

MilkyIQ replied:
afaict that map wasn't listed in the official rankings

Pachiru replied:
What do you mean by "official ranking"?

zeusminus replied:
Those mapsets were listed in the [Beatmap Charts](https://osu.ppy.sh/rankings/osu/charts?spotlight=50) of that time. Unfortunately, Neil's Bad Apple couldn't make it there.

Pachiru replied:
Ah okay my bad I thought it was like all the maps that were ranked on that period

TPGPL reviewed (approved):
lets go
